x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/search/WeightedTokensQueryBuilder.java

```diff
@@ -75,6 +70,14 @@ public TokenPruningConfig getTokenPruningConfig() {
         return tokenPruningConfig;
     }
 
+    public boolean shouldPruneTokens() {
```

Comment:
Looks like `shouldPruneTokens` and `getTokenPruningConfig` methods are not used anywhere.

Reply:
I can remove, thanks!